Neos.Utility.Schema/Classes/TYPO3/Flow/Utility/SchemaValidator.php

```diff
@@ -577,7 +577,8 @@ protected function validateAnyType($value, array $schema)
     protected function createError($expectation, $value = null)
     {
         if ($value !== null) {
-            $error = new \TYPO3\Flow\Error\Error('expected: %s found: %s', 1328557141, array($expectation, $value), 'Validation Error');
+            $valueInfo = $this->getValueInfo($value);
```

Comment:
With my other comment in place, this is also more sensible, as instead of leading to a call flow like getValueInfo('type=' . getValueInfo($value)), it would be sth. like getValueInfo('type=' . getValueType($value)).